src/main/java/com/theironyard/PurchasesController.java

```diff
@@ -0,0 +1,90 @@
+package com.theironyard;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import javax.annotation.PostConstruct;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Created by vasantia on 7/20/16.
+ */
+
+@Controller
+public class PurchasesController {
+
+    @Autowired
+    CustomerRepository customers;
+
+    @Autowired
+    PurchaseRepository purchases;
+
+    @PostConstruct
+    public void init() {
+
+        if (customers.count() < 1) {
+
+            File customerFile = new File("customers.csv");
+            Scanner scanner;
+            try {
+                scanner = new Scanner(customerFile);
+            } catch (FileNotFoundException e) {
+                e.printStackTrace();
+                return;
+            }
+            scanner.hasNext();
```

Comment:
This is incorrect.  If you are trying to get rid of the header line this simply returns a `boolean` of whether or not the scanner has another value.